docs/guides/blocks/cards.md

```diff
@@ -0,0 +1,205 @@
+
+The properties displayed in the subtitle are determined via the `subtitleProperties` field of the schema, which lists the subset of properties to show and the order to show them.
+
+```ts
+const ProductSchema = coda.makeObjectSchema({
+  properties: {
+    // ...
+    quantity: { type: coda.ValueType.Number },
+    price: { type: coda.ValueType.Number, codaType: coda.ValueHintType.Currency },
+  },
+  // ...
+  subtitleProperties: ["quantity", "price"],
+});
+```
+
+The subtitle shows a label for each property, which defaults to `{Name}: {Value}`. You can customize the label for a property, such as removing the property name, in the property definition. See the [Schemas guide][schemas_property_labels] for more information.
+
+
+### Body
+
+The card body can include a snippet of content, which appears under the title (and subtitle if defined).
+
+TODO: Image highlighting card snippet
+
+The snippet is meant to contain a limited amount of text, although there is no size limit enforced. Which property's content to use for the snippet is defined by the field `snippetProperty`, and it can only refer to properties of type `String` or `Array` of `String`. These properties can contain rich text, such as [Markdown][data_types_markdown] and [HTML][data_types_html].
+
+```ts
+const ProductSchema = coda.makeObjectSchema({
+  properties: {
+    // ...
+    description: { type: coda.ValueType.String },
+  },
+  // ...
+  snippetProperty: "description",
+});
+```
+
+
+### Image
+
+The card can include an image, which appears to the left of the other content.
+
+TODO: Image highlighting card image
+
+Which property's content to use for the image is defined by the field `imageProperty`, and it can only refer to properties of type `String` with a hint of either `ImageReference` or `ImageAttachment`.
+
+```ts
+const ProductSchema = coda.makeObjectSchema({
+  properties: {
+    // ...
+    photo: { type: coda.ValueType.String, codaType: coda.ValueHintType.ImageReference },
+  },
+  // ...
+  imageProperty: "photo",
+});
+```
+
+
+### Link
+
+The card can include a link, which will be opened when the card is clicked. The domain of the link is also shown at the bottom of the card.
+
+TODO: Image highlighting card link
+
+Which property's content to use for the link is defined by the field `linkProperty`, and it can only refer to properties of type `String` with a hint of `Url`.
+
+```ts
+const ProductSchema = coda.makeObjectSchema({
+  properties: {
+    // ...
+    websiteLink: { type: coda.ValueType.String, codaType: coda.ValueHintType.Url },
+  },
+  // ...
+  linkProperty: "websiteLink",
+});
+```
+
+
+## Upgrade a link to a card
+
+One of the most common uses for cards is to display information about an item in an external application. For instance, to preview the details of a Slack message or Google Drive file. Coda provides an easy entry point to this functionality when a user pastes a link into the doc by offering to upgrade it into a card.
```

Comment:
I wonder if it's worth highlighting that the "Card" option in the link format menu is totally different from a pack card.